packages/docs/docs/docs/zeptoken.md

```diff
@@ -0,0 +1,38 @@
+---
+id: zeptoken
+title: The ZEPToken
+sidebar_label: The ZEPToken
+---
+
+ZeppelinOS' native ZEP token is used to incentivize the creation and maintenance of a healthy ecosystem of EVM packages. It does so via a series of mechanisms that are detailed in the following sections of this article.
+
+## Compatibility standards
```

Comment:
I'd move this section to the end, and merge it with the private beta. This way, after mentioning that the ZEP token is managed via TPL, we can explain that in order to be accepted into the TPL jursidiction the user can apply to the private beta.